NDTensors/src/dense/tensoralgebra/contract.jl

```diff
@@ -360,7 +361,8 @@ function _contract!(
   tB = 'N'
   if props.permuteB
     #@timeit_debug timer "_contract!: permutedims B" begin
-    @strided Bp = permutedims(BT, props.PB)
+    #@strided 
+    Bp = permutedims(BT, props.PB)
```

Comment:
Reminder to make `NDTensors.permutedims` that dispatches on `Array` or `CuArray` to call `@strided` on `Array`.